### ISSUE TITLE
trill extension line vertical placement

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#919]
Trill extension lines are placed incorrectly when there is an accidental above the `tr` mark.  This is because the write pointer remains at the accidental when writing the line, but instead the write pointer should be set back to the `tr` mark for the correct vertical placement.

Example:

<img width="1381" alt="screen shot 2018-07-21 at 1 40 25 pm" src="https://user-images.githubusercontent.com/3487289/43035205-c75d60b8-8ceb-11e8-98a9-b4a50258f77a.png">

The first trill on each line is correct, but in the following measures the line is too high and aligns with the accidental rather than the `tr` symbol.

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-07-21T13:40:13" version="2.0.0-dev-a8ecf52-dirty">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001990582307">
                <score xml:id="score-0000000069097136">
                    <scoreDef xml:id="scoredef-0000000584169592" midi.bpm="400">
                        <staffGrp xml:id="staffgrp-0000000297219436">
                            <staffDef xml:id="staffdef-0000000937306916" clef.shape="G" clef.line="2" n="1" lines="5">
                                <label xml:id="label-0000000525360728" />
                            </staffDef>
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-L1F1">
                        <measure xml:id="measure-L2">
                            <staff xml:id="staff-L2F1N1" n="1">
                                <layer xml:id="layer-L2F1N1" n="1">
                                    <note xml:id="note-L3F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L3F1" staff="1" startid="#note-L3F1" tstamp2="0m+5.0" />
                        </measure>
                        <measure xml:id="measure-L4">
                            <staff xml:id="staff-L4F1N1" n="1">
                                <layer xml:id="layer-L4F1N1" n="1">
                                    <note xml:id="note-L6F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L6F1" staff="1" startid="#note-L6F1" tstamp2="0m+5.0" accidupper="f" />
                        </measure>
                        <measure xml:id="measure-L7">
                            <staff xml:id="staff-L7F1N1" n="1">
                                <layer xml:id="layer-L7F1N1" n="1">
                                    <note xml:id="note-L9F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L9F1" staff="1" startid="#note-L9F1" tstamp2="0m+5.0" accidupper="ff" />
                        </measure>
                        <measure xml:id="measure-L10">
                            <staff xml:id="staff-L10F1N1" n="1">
                                <layer xml:id="layer-L10F1N1" n="1">
                                    <note xml:id="note-L12F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L12F1" staff="1" startid="#note-L12F1" tstamp2="0m+5.0" accidupper="s" />
                        </measure>
                        <measure xml:id="measure-L13">
                            <staff xml:id="staff-L13F1N1" n="1">
                                <layer xml:id="layer-L13F1N1" n="1">
                                    <note xml:id="note-L15F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L15F1" staff="1" startid="#note-L15F1" tstamp2="0m+5.0" accidupper="x" />
                        </measure>
                        <measure xml:id="measure-L16" right="dbl">
                            <staff xml:id="staff-L16F1N1" n="1">
                                <layer xml:id="layer-L16F1N1" n="1">
                                    <note xml:id="note-L18F1" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L18F1" staff="1" startid="#note-L18F1" tstamp2="0m+5.0" accidupper="n" />
                        </measure>
                        <measure xml:id="measure-L19">
                            <staff xml:id="staff-L19F1N1" n="1">
                                <layer xml:id="layer-L19F1N1" n="1">
                                    <note xml:id="note-L20F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L20F1" staff="1" startid="#note-L20F1" tstamp2="0m+5.0" />
                        </measure>
                        <measure xml:id="measure-L21">
                            <staff xml:id="staff-L21F1N1" n="1">
                                <layer xml:id="layer-L21F1N1" n="1">
                                    <note xml:id="note-L23F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L23F1" staff="1" startid="#note-L23F1" tstamp2="0m+5.0" accidupper="f" />
                        </measure>
                        <measure xml:id="measure-L24">
                            <staff xml:id="staff-L24F1N1" n="1">
                                <layer xml:id="layer-L24F1N1" n="1">
                                    <note xml:id="note-L26F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L26F1" staff="1" startid="#note-L26F1" tstamp2="0m+5.0" accidupper="ff" />
                        </measure>
                        <measure xml:id="measure-L27">
                            <staff xml:id="staff-L27F1N1" n="1">
                                <layer xml:id="layer-L27F1N1" n="1">
                                    <note xml:id="note-L29F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L29F1" staff="1" startid="#note-L29F1" tstamp2="0m+5.0" accidupper="s" />
                        </measure>
                        <measure xml:id="measure-L30">
                            <staff xml:id="staff-L30F1N1" n="1">
                                <layer xml:id="layer-L30F1N1" n="1">
                                    <note xml:id="note-L32F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L32F1" staff="1" startid="#note-L32F1" tstamp2="0m+5.0" accidupper="x" />
                        </measure>
                        <measure xml:id="measure-L33">
                            <staff xml:id="staff-L33F1N1" n="1">
                                <layer xml:id="layer-L33F1N1" n="1">
                                    <note xml:id="note-L35F1" dur="1" oct="5" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <trill xml:id="trill-L35F1" staff="1" startid="#note-L35F1" tstamp2="0m+5.0" accidupper="n" />
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed with 3c8c818d042bb0a7a830de571cbdd6498f2d9ace

**Comment by craigsapp** [comment]
Looks good:

<img width="1354" alt="screen shot 2018-07-23 at 6 44 27 pm" src="https://user-images.githubusercontent.com/3487289/43090654-4e0f0658-8ea8-11e8-9016-1c28932c6561.png">
